buildtest/buildsystem/base.py

```diff
@@ -457,13 +463,13 @@ def run_tests(self, testfile):
         if command.returncode == 0:
             print(
                 "{:<30} {:<30} {:<30} {:<30}".format(
```

Comment:
Name is also something that I’d rather not rename like this

Reply:
i fixed this in commit https://github.com/HPC-buildtest/buildtest-framework/pull/270/commits/e052d3d83da9ca2200bbe16ab4c6c43208b4d27c